### PR TITLE
AVRO-3275: Clarify specification on naming

### PR DESCRIPTION
**Pull request by opwvhk**
Mirror of PR apache/avro#1439

### CODE REVIEW

**Review by martin-g — APPROVED** [review]
Thank you, @opwvhk !
I will merge it once the upstream PR is merged!

File: content/en/docs/1.11.0/Specification/_index.md [hunk 1]
```diff
@@ -172,11 +172,59 @@ The null namespace may not be used in a dot-separated sequence of names. So the
...
+        "type": "enum",
+        "name": "Simple",
+        "doc": "A simple name (attribute) and no namespace attribute: inherit the null namespace of the enclosing type 'Example'. The fullname is 'Simple'.",
+        "symbols": ["a", "b"]
+      }
+    }, {
+      "name": "explicitNamespace",
+      "type": {
+        "type": "fixed",
+        "name": "simple",
```
Comment (RyanSkraba) [inline content/en/docs/1.11.0/Specification/_index.md:194]:
```suggestion
        "name": "Simple",
```

File: content/en/docs/1.11.0/Specification/_index.md [hunk 2]
```diff
@@ -172,11 +172,59 @@ The null namespace may not be used in a dot-separated sequence of names. So the
...
+        "namespace": "explicit",
+        "doc": "A simple name (attribute) and a namespace (attribute); the fullname is 'explicit.Simple' (this is a different type than of the 'inheritNull' field)",
```
Comment (RyanSkraba) [inline content/en/docs/1.11.0/Specification/_index.md:196]:
```suggestion
        "doc": "A simple name (attribute) and a namespace (attribute); the fullname is 'explicit.Simple' (this is a different type than of the 'inheritNull' field).",
```

So nitpicky for consistency :D

**Review by RyanSkraba — APPROVED** [review]

**Comment by opwvhk** [comment]
As the changes are on the Avro website, this PR is no longer needed.

**Pull request closed without merging**